ee/clickhouse/views/element.py

```diff
@@ -0,0 +1,60 @@
+from rest_framework import authentication, request, response, serializers, viewsets
+from rest_framework.decorators import action
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.models.element import chain_to_elements
+from ee.clickhouse.models.property import parse_prop_clauses
+from ee.clickhouse.queries.util import parse_timestamps
+from ee.clickhouse.sql.element import GET_ELEMENTS, GET_VALUES
+from posthog.api.element import ElementSerializer, ElementViewSet
+from posthog.models.filter import Filter
+
+
+class ClickhouseElement(ElementViewSet):
```

Comment:
I'm not sure if ElementViewSet is used another way besides /stats and /values but if there's a call to something like list elements (api/elements) it's going to default to PG elements which are not maintained now on.

Reply:
Yeah I checked this, we don't actually call it anywhere. Would be good to get rid of it but not urgent.